src/konnektor/network_planners/generators/minimal_spanning_tree_network_generator.py

```diff
@@ -21,29 +21,29 @@ def __init__(
         _initial_edge_lister: NetworkGenerator = None,
     ):
         """
-        The `MinimalSpanningTreeNetworkGenerator`, builds an minimal spanning tree (MST) network for a given set of `Component` s.\
-        The `Transformation` s of the Network are represented by an `AtomMapping` s, which are scored by a `AtomMappingScorer`.
+        The ``MinimalSpanningTreeNetworkGenerator``, builds an minimal spanning tree (MST) network for a given set of ``Component``/s.\
+        The ``Transformation`` s of the network are represented by an ``AtomMapping`` s, which are scored by a ``AtomMappingScorer``.
 
         For the MST algorithm, the Kruskal Algorithm is used.
 
-        The MST algorithm gives the optimal graph score possible and the minimal required set of `Transformations`.
+        The MST algorithm gives the optimal graph score possible and the minimal required set of ``Transformations``.
         This makes the  MST Network very efficient. However, the MST is not very robust, in case of one failing
-        `Transformation`, the Network is immediatly disconnected.
-        The disconnectivity will translate to a loss of `Component` s in the final FE Network.
+        ``Transformation``, the network is immediately disconnected.
+        The disconnectivity will translate to a loss of ``Component``/s in the final FE Network.
```

Comment:
A question here for something we should do in another issue - does it make sense for us to be talking in terms of Transformations and Components?

My understanding is that Konnektor doesn't know about them, it just knows about ligands and edges.

This is useful info at the openfe / gufe level, but I wonder if it shouldn't be excluded from here?